src/platform/packages/shared/kbn-esql-ast/src/parser/__tests__/fuse.test.ts

```diff
@@ -53,16 +53,5 @@ describe('FUSE', () => {
 
       expect(errors.length > 0).toBe(true);
     });
-
```

Comment:
Fuse accepts arguments now so this test should not fail. We will take care of the fuse new syntax in another PR